LICENSE.md

```diff
@@ -19,14 +22,3 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
-
-The source implementation was originally copied from that found in `AbstractAlgebra.jl`
```

Comment:
I think we should probably keep this? Given that it says "Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer."

maybe there are other licenses we should copy in as well?

Reply:
I changed this based on my understand of @fingolfin's comments here https://github.com/Nemocas/AbstractAlgebra.jl/issues/1666#issuecomment-2052081190, that

> so it would be fine to have just the MIT license in your repository

Reply:
ah ok, I missed that bit